assets/findify-autocomplete.js

```diff
@@ -110,41 +110,51 @@ const initAutocomplete = () => {
           document.querySelector('.findify-autocomplete').className += ' hidden';
         }
     };
-
+  
     const openAutocomplete = () => {
         const autocompleteClassName = document.querySelector('.findify-autocomplete').className;
         document.querySelector('.findify-autocomplete').className = autocompleteClassName.replace(' hidden', '');
         document.querySelector('.findify-close-autocomplete')?.addEventListener("click", (e) => closeAutocomplete(e));
-        setViewAll();
+        setTitle();
     };
-
+  
     const onScrollListener = () => {
         document.addEventListener('scroll', () => {
           if (document.querySelector('.findify-autocomplete').className.includes('hidden') == false)
             setAutocompletePosition();
         })
     };
-
+  
     const setViewAll = () => {
         const viewAll = document.querySelector('.findify-view-all')?.href;
         const q = document.querySelector(selector).value;
         document.querySelectorAll('.findify-view-all').forEach(i => {
           i.href = `${viewAll}${q}`;
           if (q) { 
-              if (isMobile) {    
-                  i.classList.add('findify-view-all-with-query')
-                  i.textContent = 'View all results for';
-                  i.innerHTML += `<span class="findify-components-autocomplete--tip__highlight"><br>"${q}"</span>`
-              } else {
-                i.textContent = 'View all results for';
-                i.innerHTML += `<span class="findify-components-autocomplete--tip__highlight">"${q}"</span>`
-              }
+              if (isMobile) i.classList.add('findify-view-all-with-query');
+              i.textContent = 'View all results for';
+              i.innerHTML += `<span class="findify-components-autocomplete--tip__highlight">"${q}"</span>`;
           } else {
             i.textContent = 'View all';
           }
         })
     };
-
+  
+    const setTitle = () => {
+      const q = document.querySelector(selector).value;
+      const suggestions = document.querySelector('.suggestions-wrapper h3');
+      const products = document.querySelector('.products-wrapper h3');
+      
+      if (q) {
+        suggestions.textContent = 'Search suggestions';
+        products.textContent = 'Product matches';
+      } else {
+        suggestions.textContent = 'Trending searches';
+        products.textContent = 'Trending products';
+      }
+      return;
+    };
```

Comment:
All these above is JS to manipulate text.
Later surely we will take in consideration using Shopify Translations.

The ideal approach to this is within liquid to use:
```
{% if q == undefined %} (or something in this line)
this text
{% else %}
another text
{%endif%}
```